src/order/basic.lean

```diff
@@ -174,6 +174,12 @@ def strict_mono_decr_on [has_lt α] [has_lt β] (f : α → β) (t : set α) : P
 def order_dual (α : Type*) : Type* := α
 
 namespace order_dual
+
+/-- Recursor for `order_dual α`. -/
+@[elab_as_eliminator]
+def dual_rec {C : order_dual α → Sort*} (h₂ : Π (a : α), C (a : order_dual α)) :
```

Comment:
This statement feels like it should be
```suggestion
def dual_rec {C : order_dual α → Sort*} (h₂ : Π (a : α), C (to_dual a)) :
```
but `to_dual` is in another file.

Reply:
I think move it to that file. `to_dual` is in another file because putting it here created import loops (#4758).